template/index.html

```diff
@@ -1,38 +1,44 @@
 <!DOCTYPE html>
-{% from "macros/commons.html" import getApiVersion with context %}
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
+  <head>
```

Comment:
Same as above, you should probably convert this template into a React template.

Reply:
You can see in commits that I had a template written in React, but due to this asyncapi/generator#521 I wanna stay with Nunjucks. When mentioned issue will be fixed, rewritten to React will be very easy :)